db/migrate/20221110020049_make_recipient_case_id_optional.rb

```diff
@@ -0,0 +1,8 @@
+class MakeRecipientCaseIdOptional < ActiveRecord::Migration[7.0]
+  def change
+
+    change_column_null :recipients, :message_batch_id, true
```

Comment:
Why is this needed?

Reply:
It's a consequence of moving status out of SmsMessage and onto Recipient, which means that we now need a Recipient to properly send a message and keep track of its delivery. So now not all Recipients are part of a message batch: Some now only exist to send a response to a consent change message. I don't like the current polymorphic feel of Recipient - maybe we can think of something better.